kernel/public/test-scenes/-101.105.avatarshape/mannequin.ts

```diff
@@ -0,0 +1,220 @@
+import { engine, Entity, Transform, AvatarShape, Vector3, Vector2, Color4, executeTask, log, BoxShape, Material, OnPointerDown, ActionButton } from 'decentraland-ecs/src'
+
+const PREVIEW_DEBUGGING = false;
+
+const sceneCoords = new Vector2(-149, -144);
+const parcelSize = 16;
+
+const blockConfig = {
+  offset: new Vector3(0, 1, 0),
+  scale: new Vector3(1, 2, 1),
+  color: new Color4(0.69, 0.85, 0.96, 0.2),
+};
+
+const avatarConfig = {
+  offset: new Vector3(0, 0.01, 0),
+  nameTag: "Mannequin",
+};
+
+let wearablesCatalog: WearableCatalog;
+
+export class Mannequin {
+  private mainEntity: Entity | any = null;
+  private mainTransform: Transform | any = null;
+
+  private avatar: MannequinAvatar | any = null;
+  private avatarTransform: Transform | any = null;
+
+  private position: Vector3 = Vector3.Zero();
+  private userId: string;
+
+  constructor(userId: string) {
+    this.userId = userId;
+    // const wearablesUrl = `https://dcl-wearables-dev.now.sh/expected.json`;
```

Comment:
we should remove this commented line

Reply:
Removed that test scene as it wasn't finished and we didn't need it in the end